docs/guides/dynamo_deploy/dynamo_cloud.md

```diff
@@ -65,6 +65,19 @@ helm fetch https://helm.ngc.nvidia.com/nvidia/ai-dynamo/charts/dynamo-platform-$
 helm install dynamo-platform dynamo-platform-${RELEASE_VERSION}.tgz --namespace ${NAMESPACE}
 ```
 
+> [!TIP]
+> By default, Model Express Server is NOT installed. You can enable it by setting the following flag in the helm install command:
+
+```bash
+--set "global.model-express.enabled=true"
+```
+
+> If an existing Model Express Server is running, you can set the modelExpressURL to the existing server's URL.
+```bash
+--set "dynamo-operator.modelExpressURL=http://model-express-server.model-express.svc.cluster.local:8080"
+```
+
```

Comment:
Nit: can you go into a bit of detail on what happens if the user tries to set the modelExpressUrl but did not enable it?